neuronunit/models/lems.py

```diff
@@ -206,4 +206,13 @@ def has_pulse_generator(self, tree=None):
     def temp_dir(self):
         if not hasattr(self, '_temp_dir'):
             self._temp_dir = TemporaryDirectory()
-        return self._temp_dir
\ No newline at end of file
+        return self._temp_dir
+
+    def get_state_variables(self):
+        lems_tree = etree.parse(self.lems_file_path)
```

Comment:
Can you add a docstring here explaining what it does?  You can use the Google-style docstring format: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html.  I know that most of the library does not have them yet, but that is a work in progress (see the other open PRs).

Reply:
Done!